openedx/core/djangoapps/edx_global_analytics/utils.py

```diff
@@ -192,3 +192,35 @@ def platform_coordinates(city_name):
     Module `pytz` also has list of cities with `pytz.all_timezones`.
     """
     return get_coordinates_by_platform_city_name(city_name) or get_coordinates_by_ip()
+
+
+def request_exception_handler_with_logger(function):
+    """
+    Request Exception decorator. Logs error message if it exists.
+    """
+    def request_exception_wrapper(*args, **kwargs):
+        """
+        Decorator wrapper.
+        """
+        try:
+            return function(*args, **kwargs)
+        except requests.RequestException as error:
+            logger.exception(error.message)
+            return
```

Comment:
> How celery behave if exception is happened in the task? If I not mistaked, it logs it, and task is stopped for the current run. In this case we can just leave it w/o wrapper. What you think?

Task will log exception, but won' shut down.
I will add explanation to docstring.